File: Lista/quest10/metodos.go
package main

import "fmt"

type Calculadora struct{}

func (c Calculadora) SomarDois(a, b int) int {
	return a + b
}

func (c Calculadora) SomarTres(a, b, c int) int {
	return a + b + c
}

func main() {
	calc := Calculadora{}

	fmt.Println("Soma de 2 números:", calc.SomarDois(5, 10))
	fmt.Println("Soma de 3 números:", calc.SomarTres(5, 10, 15))
}

File: Lista/quest10/metodos.py
class Calculadora:
    def somar_dois(self, a, b):
        return a + b

    def somar_tres(self, a, b, c):
        return a + b + c

if __name__ == "__main__":
    calc = Calculadora()

    print("Soma de 2 números:", calc.somar_dois(5, 10))
    print("Soma de 3 números:", calc.somar_tres(5, 10, 15))
